### ISSUE TITLE
Cannot import flask login, issue with Werzeug 3.0.0

### ISSUE THREAD

**Issue by MatteoRagni** [maxcountryman/flask-login#779]
**Describe the bug**
Cannot import flask login, issue with Werzeug 3.0.0

**To Reproduce**

1. Create an empty virtualenv

   ```
   python3 -m venv .venv && . .venv/bin/activate
   ```

2. Install flask login with pip (current version is 0.6.2 that installs Werzeug 3.0.0

   ```
   pip install flask-login
   ```

3. Try to import Flask Login

   ```
   python
   >>> import flask_login
   ```

The current traceback is:

```
Python 3.10.12 (main, Jun 11 2023, 05:26:28) [GCC 11.4.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import flask_login
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/tmp/.venv/lib/python3.10/site-packages/flask_login/__init__.py", line 12, in <module>
    from .login_manager import LoginManager
  File "/tmp/.venv/lib/python3.10/site-packages/flask_login/login_manager.py", line 33, in <module>
    from .utils import _create_identifier
  File "/tmp/.venv/lib/python3.10/site-packages/flask_login/utils.py", line 14, in <module>
    from werkzeug.urls import url_decode
ImportError: cannot import name 'url_decode' from 'werkzeug.urls' (/tmp/.venv/lib/python3.10/site-packages/werkzeug/urls.py)
```

Pip freeze for versions reference

```
blinker==1.6.2
click==8.1.7
Flask==3.0.0
Flask-Login==0.6.2
itsdangerous==2.1.2
Jinja2==3.1.2
MarkupSafe==2.1.3
Werkzeug==3.0.0
```
**Expected behavior**
A correct import

**Desktop (please complete the following information):**
 - OS: Ubuntu 22.04 LTS

**Comment by davidism** [comment]
Please search for duplicates first.

**Comment by davidism** [comment]
Duplicate of #744, fixed by #746. A volunteer maintainer will work on making a release when they have time available.
